README.md

```diff
@@ -59,19 +59,21 @@ Your local port `9999` forwards now to the remote host on address `127.0.0.1` an
 - `dataPath` — path to folder with network data (default: `~/.lotion/networks/<network>—<networkdId>`)
 
 ### Config file options
+```
   "bridgeAddr": "0x7b8342412883f4b34f335d4e1391ec190eb887ca",
   "operatorAddr": "0x9b83018de826c0343af6e682c24e7c91a421755c",
   "exitHandlerAddr": "0x0b32eb1aaa9b0804852f4fe1b2e6100edb4533d8",
   "rootNetwork": "https://rinkeby.infura.io",
   "rootNetworkId": 4,
   "network": "leap-testnet-gamma",
   "networkId": 1341,
+```
 
 - `bridgeAddr` — leap [Bridge](https://github.com/leapdao/leap-contracts) contract address
 - `operatorAddr` — leap [Operator](https://github.com/leapdao/leap-contracts) contract address
 - `exitHandlerAddr` — leap [ExitHandler](https://github.com/leapdao/leap-contracts) contract address
-- `rootNetwork` — *DEPRECATED. Use `rootNetworkId` instead.* Ethereum provider url (e.g. `https://rinkeby.infura.io`)
-- `rootNetworkId` — root network id (see [src/utils/networks.js](https://github.com/leapdao/leap-node/blob/master/src/utils/networks.js),
+- `rootNetwork` — Ethereum provider URL (e.g. `https://rinkeby.unfura.io`)
```

Comment:
😂

Reply:
Oh I think I mistyped it for real :grimacing: